src/batch-transaction.ts

```diff
@@ -321,7 +339,7 @@ class BatchTransaction extends Snapshot {
             }
 
             span.end();
-            callback(err, partitions, resp);
+            cb!(err, partitions, resp);
```

Comment:
Wny cb , why not callback ?

Reply:
it can be callback as well,  just to make code cleaner I used cb here

we already are using such idiomatic in our existing code as well [ref](https://github.com/googleapis/nodejs-spanner/blob/main/src/database.ts#L2886)

Reply:
got it.